src/tilemaker.cpp

```diff
@@ -119,6 +120,16 @@ void WriteFileMetadata(rapidjson::Document const &jsonConfig, SharedData const &
 	fclose(fp);
 }
 
+double bboxElementFromStr(const string& number) {
+	char* endptr;
+	double num = strtod(number.c_str(), &endptr);
```

Comment:
Use a lexical_cast for this: 
https://www.geeksforgeeks.org/boost-lexical_cast-in-cpp/